lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_create_avd_action.rb

```diff
@@ -0,0 +1,90 @@
+
+        # Create the AVD for device, API and system image we need
+        helper.create_avd(
+          api: api_level,
+          device: device_model,
+          system_image: system_image,
+          name: avd_name,
+          sdcard: sdcard
+        )
+      end
+
+      #####################################################
+      # @!group Documentation
+      #####################################################
+
+      def self.description
+        'Creates a new Android Virtual Device (AVD) for a specific device model and API level'
+      end
+
+      def self.details
+        <<~DESC
+          Creates a new Android Virtual Device (AVD) for a specific device model and API level.
+          By default, it also installs the necessary system image (using `sdkmanager`) if needed before creating the AVD
+        DESC
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(key: :device_model,
+                                       env_name: 'FL_ANDROID_CREATE_AVD_DEVICE_MODEL',
+                                       description: 'The device model code to use to create the AVD. Valid values can be found using `avdmanager list devices`',
+                                       type: String,
+                                       optional: false),
+          FastlaneCore::ConfigItem.new(key: :api_level,
+                                       env_name: 'FL_ANDROID_CREATE_AVD_API_LEVEL',
+                                       description: 'The API level to use to install the necessary system-image and create the AVD',
+                                       type: Integer,
+                                       optional: false),
+          FastlaneCore::ConfigItem.new(key: :avd_name,
+                                       env_name: 'FL_ANDROID_CREATE_AVD_AVD_NAME',
+                                       description: 'The name to give to the created AVD. If not provided, will be derived from device model and API level',
+                                       type: String,
+                                       optional: true,
+                                       default_value: nil),
+          FastlaneCore::ConfigItem.new(key: :sdcard,
+                                       env_name: 'FL_ANDROID_CREATE_AVD_SDCARD',
+                                       description: 'The size of the SD card to use for the AVD',
+                                       type: String,
+                                       optional: true,
+                                       default_value: '512M'),
+          FastlaneCore::ConfigItem.new(key: :system_image,
+                                       env_name: 'FL_ANDROID_CREATE_AVD_SYSTEM_IMAGE',
+                                       description: 'The system image to use (as used/listed by `sdkmanager`). Defaults to the appropriate system image given the API level requested and the current machine\'s architecture',
```

Comment:
No reason really. I think it's a matter of finding balance as to when to switch to double-quotes when it's only to avoid escaping apostrophes in the String. If it's only one to be escaped, it's probably fine to keep single-quotes to avoid needing string interpolation to kick in for nought, when it's more than one, it's easier to switch to double-quotes to avoid escaping each one? But I don't have a strict rule for it tbh.

Another alternative would be to use alternate delimiters that still don't do string interpolation — since Ruby supports multiple string delimiters, interpolated or not. For example `%(…)` instead of `'…'` is a good choice when it comes to not needing interpolation but still wanting to avoid escaping.
```suggestion
                                       description: %(The system image to use (as used/listed by `sdkmanager`). Defaults to the appropriate system image given the API level requested and the current machine's architecture),
```